### ISSUE TITLE
Merge lines that are next to each other with no text into one big field

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/FormFyxer#46]
When the form has multiple lines in a row but it's one field--can we use heuristics to help people catch this?

**Comment by BryceStevenWilley** [comment]
Fixed in #55, will close when it's merged. It's only a first pass, but there's not a lot of other low hanging fruit here. Notably, a lot of multiple lines in a row have a partial line on the first row, i.e.

```
Why is Ladybird iconic? _________
__________________________________
__________________________________
```

There's no PDF field (to my knowledge) that can have all of the lines in the same field; you have to split off the first line into a separate field, or just skip it.

**Comment by nonprofittechy** [comment]
Yeah, my practice when you have a field like that (with a short first line) is to just skip it. I'm inclined to have doing that automatically be on our roadmap, but it's not a big manual cleanup task either.
